client/src/components/Tests/TopDishes2.test.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { shallow } from "enzyme";
 
 // component
-import TopDishes from "./TopDishes";
+import TopDishes from "../TopDishes";
```

Comment:
Not a part of this pull request, but you may be able to tweak some of your console logs to be expect/other assertions.

Reply:
In particular, you can use 'mount' instead of 'shallow' to find nested children in a DOM element